### ISSUE TITLE
How to use the scenarios in scenario_runner to reinforcement learning directly？Is there an example？

### ISSUE THREAD

**Issue by yhh-IV** [carla-simulator/scenario_runner#1042]
The official tutorial gives an example, wherein two files "scenario_runner" and "manual_control" are ran in two terminals, respectively. 
I want to load scenarios in scenario runner for RL, and there are two challenges: 

1. How can I integrate the scenario_runner class into the training file, therefore we can reset the scenario easily for the next training episode.
2. In the official tutorial, the program always tries to kill the already dead actor after completing the scenarios, how to avoid this since it could prevent the initiation of the next episode in RL.

In summary, would anyone provide a tutorial to us to help establish a RL training environment through scenario runner. Many thanks for the help!!!
